app/views/two_factor_authentication/backup_code_verification/show.html.erb

```diff
@@ -8,8 +8,10 @@
   <%= t('two_factor_authentication.backup_code_prompt') %>
 </p>
 
-<%= validated_form_for(@backup_code_form, url: login_two_factor_backup_code_path,
-                       html: { autocomplete: 'off', method: :post }) do |f| %>
+<%= validated_form_for(
+      @backup_code_form, url: login_two_factor_backup_code_path,
+                         html: { autocomplete: 'off', method: :post }
```

Comment:
this is very weird indentation... if it were me, I would do:
```suggestion
      @backup_code_form,
      url: login_two_factor_backup_code_path,
      html: { autocomplete: 'off', method: :post }
```

Reply:
Yeah, I think the linter would be fine with that as well. The indentation it's proposing is based on the idea of aligning all hash properties to the beginning of the first hash property, which would work both as it auto-corrected, and also as you suggest.

Reply:
Needed a trailing comma, but otherwise linter happy 👍  270c8268a